scripts/variantstore/wdl/extract/ngs_cohort_extract.py

```diff
@@ -229,27 +229,24 @@ def do_extract(fq_pet_vet_dataset,
                min_variant_samples,
                fq_sample_mapping_table
               ):
-  try:  
       
-    global client
-    client = bigquery.Client(project=query_project, 
-                             default_query_job_config=QueryJobConfig(labels={ "id" : f"test_cohort_export_{output_table_prefix}"}, priority="INTERACTIVE", use_query_cache=False ))
+  global client
+  client = bigquery.Client(project=query_project,
+                           default_query_job_config=QueryJobConfig(labels={ "id" : f"test_cohort_export_{output_table_prefix}"}, priority="INTERACTIVE", use_query_cache=False ))
 
-    ## TODO -- provide a cmdline arg to override this (so we can simulat smaller datasets)
-    global PET_VET_TABLE_COUNT
-    PET_VET_TABLE_COUNT = max_tables
-    print(f"Using {PET_VET_TABLE_COUNT} PET tables in {fq_pet_vet_dataset}...")
+  ## TODO -- provide a cmdline arg to override this (so we can simulat smaller datasets)
+  global PET_VET_TABLE_COUNT
+  PET_VET_TABLE_COUNT = max_tables
+  print(f"Using {PET_VET_TABLE_COUNT} PET tables in {fq_pet_vet_dataset}...")
 
-    cohort = get_all_samples(fq_cohort_sample_names, fq_sample_mapping_table)
-    print(f"Discovered {len(cohort)} samples in {fq_cohort_sample_names}...")
+  cohort = get_all_samples(fq_cohort_sample_names, fq_sample_mapping_table)
+  print(f"Discovered {len(cohort)} samples in {fq_cohort_sample_names}...")
 
-    make_new_vet_union_all(fq_pet_vet_dataset, fq_temp_table_dataset, cohort)
+  make_new_vet_union_all(fq_pet_vet_dataset, fq_temp_table_dataset, cohort)
 
-    create_position_table(fq_temp_table_dataset, min_variant_samples)  
-    make_new_pet_union_all(fq_pet_vet_dataset, fq_temp_table_dataset, cohort)
-    populate_final_extract_table(fq_temp_table_dataset, fq_destination_dataset, destination_table, fq_sample_mapping_table)
-  except Exception as err:
```

Comment:
I think this will just eat the exception, but then continue on to print the "Final cohort extract written to..." error message (which isn't true?)

Reply:
the try/catch is being removed so it will no longer eat the exception. it seems counter intuitive to remove the try/catch but i'm not sure the best way to handle this. maybe add a comment that states exceptions are not being caught so the calling script will error out when an exception occurs. i feel like we could into a cycle otherwise where we try to add exception handling back in.

Reply:
that works!

Reply:
pushed up that change